cfme/services/catalogs/service_catalogs.py

```diff
@@ -33,47 +39,74 @@
     ])
 
 
-menu.nav.add_branch(
-    'services_catalogs',
-    {
-        'service_catalogs':
-        [
-            lambda _: accordion.click('Service Catalogs'),
-            {
-                'service_catalog':
-                [
-                    lambda ctx: accordion_tree(
-                        'All Services', ctx['catalog'], ctx['catalog_item'].name),
-                    {
-                        'order_service_catalog': lambda _: sel.click(order_button)
-                    }
-                ]
-            }
-        ]
-    }
-)
-
-
-class ServiceCatalogs(Updateable, Pretty):
+match_page = partial(match_location, title='Catalogs', controller='catalog')
+
+
+def nav_to_all():
```

Comment:
I think since there is only one class in this module, and the other catalogs classes are already in different modules in the services.catalogs package, this won't be used outside of the 'All' destination registered for ServiceCatalogs. Thus you can drop this function and do it directly in the `All` destination step. We used this when there were multiple classes in a module with 2+ `All` destinations that shared navigation steps.